apps/search/js/private_browsing.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+/* global newtab */
+
+(function(exports) {
+
+  function PrivateBrowsing() {
+    this.trigger = document.getElementById('private-window');
+    this.trigger.addEventListener('click', this);
+  }
+
+  PrivateBrowsing.prototype = {
+
+    /**
+     * General event handler.
+     * Currently dispatches a request to open a new private window
+     * from the newtab object.
+     */
+    handleEvent: function(e) {
+      newtab.requestPrivateWindow();
```

Comment:
It also seems a little weird that we have a module here that depends on a the global newtab object. Seems like the logic here (ie. listening for the button click) would be fine in newtab.js. Especially if we already have the `requestPrivateWindow` logic in there.